### ISSUE TITLE
[@pika/plugin-bundle-web] Version 0.9.0+ errors with missing dependencies

### ISSUE THREAD

**Issue by gr2m** [pikapkg/builders#110]
I've tried to upgrade https://github.com/gr2m/universal-github-app-jwt to the latest builders, but starting 0.9.0, the `@pika/plugin-bundle-web` package is starting to throw errors

```
> pika build

@pika/pack v0.5.0
Error: Cannot find module 'rollup-plugin-json'
Require stack:
- /Users/gregor/Projects/universal-github-app-jwt/node_modules/@pika/plugin-bundle-web/dist-node/index.js
- /Users/gregor/Projects/universal-github-app-jwt/node_modules/import-from/index.js
- /Users/gregor/Projects/universal-github-app-jwt/node_modules/@pika/pack/dist-node/index.js
- /Users/gregor/Projects/universal-github-app-jwt/node_modules/@pika/pack/dist-node/index.bin.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/gregor/Projects/universal-github-app-jwt/node_modules/@pika/plugin-bundle-web/dist-node/index.js:8:34)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19)
Error: Command failed with exit code 1 (EPERM): npx pika-pack
    at makeError (/Users/gregor/Projects/universal-github-app-jwt/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/Users/gregor/Projects/universal-github-app-jwt/node_modules/execa/index.js:112:26)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
    at async runExternalCommand (/Users/gregor/Projects/universal-github-app-jwt/node_modules/@pika/cli/dist-node/index.js:83:5)
    at async cli (/Users/gregor/Projects/universal-github-app-jwt/node_modules/@pika/cli/dist-node/index.js:141:43)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! universal-github-app-jwt@0.0.0-development build: `pika build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the universal-github-app-jwt@0.0.0-development build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/gregor/.npm/_logs/2020-02-13T18_54_21_270Z-debug.log
```

My config looks like this

```json
  "@pika/pack": {
    "pipeline": [
      [
        "@pika/plugin-ts-standard-pkg"
      ],
      [
        "@pika/plugin-build-node"
      ],
      [
        "@gr2m/pika-plugin-build-web"
      ],
      [
        "@pika/plugin-bundle-web",
        {
          "browser": true
        }
      ]
    ]
  },
```

I still have to use my `@gr2m/pika-plugin-build-web` fork because the pika version is not handling the "browser" key for internal modules, only for entry keys. I'll probably workaround this by creating a separate modules `browser-github-app-jwt` & `node-github-app-jwt` and then use `universal-github-app-jwt` to just do the delegation.

I do not think that my setup is related to the problems though? It looks to me like `@pika/plugin-bundle-web` is missing the `rollup-plugin-json` dependency?

Once I add it it seems to work

**Comment by FredKSchott** [comment]
Bah, thanks for the catch. That was a bad find-and-replace on my part to get rid of Rollups recent deprecation warnings on their official plugins. Updated to point to the new plugins, released as v0.9.2.

**Comment by FredKSchott** [comment]
Closing the issue, but let me know if you run into any other problems with the release
